ArduPlane/RC_Channel.cpp

```diff
@@ -443,6 +444,21 @@ bool RC_Channel_Plane::do_aux_function(const aux_func_t ch_option, const AuxSwit
         }
         break;
 
+    case AUX_FUNC::TECS_PROP_FAILED:
+        switch (ch_flag) {
+        case AuxSwitchPos::HIGH:
+            plane.TECS_controller.set_propulsion_failed_flag(true);
+            gcs().send_text(MAV_SEVERITY_INFO, "TECS: set 'propulsion failed' flag");
```

Comment:
Add to `RC_Channel::LookupTable RC_Channel::lookuptable` instead

Reply:
Ah, didn't notice those. That's much better.